samples/subsys/shell/shell_module/sample.yaml

```diff
@@ -2,13 +2,18 @@ sample:
   name: Shell Sample
 tests:
   test:
-    filter: ( CONFIG_UART_CONSOLE and CONFIG_SERIAL_SUPPORT_INTERRUPT )
+    filter: ( CONFIG_SERIAL)
```

Comment:
What is the motivation of this change (and similar one a few lines below)? "Adding shell_rtt.c to sanity check" does not seem to explain such need.

Reply:
These were old shell requirements for old shell subsusystem. I can add this change separate commit.
New shell requires only  serial to be running, legacy one was depending on UART_CONSOLE and RX interrupt.

Current shell does not use UART_CONSOLE at all and can work with or without interrupts support.

Reply:
> I can add this change in separate commit.

I think this would be good, to avoid potential confusion.

Reply:
fixed